lib/rbi/model.rb

```diff
@@ -1085,6 +1158,11 @@ def <<(param)
       @params << param
     end
 
+    sig { params(name: String, type: String).void }
```

Comment:
`<<` is the current way to use the API. It's also used in all kind of nodes to shovel children nodes.

Removing it would mean first deprecating, merging all clients, releasing etc.